library/integrations/express.ts

```diff
@@ -0,0 +1,30 @@
+import type { Express } from "express";
+import { shouldBlockRequest } from "./shouldBlockRequest";
+
+/**
+ * Calling this function will setup rate limiting and user blocking for the provided Express app.
+ * Attacks will still be blocked by Zen if you do not call this function.
+ * Execute this function as early as possible in your Express app, but after the middleware that sets the user.
+ */
+export function setupExpressIntegration(app: Express) {
+  app.use((req, res, next) => {
+    const result = shouldBlockRequest();
+
+    if (result.block) {
+      if (result.type === "ratelimited") {
+        let message = "You are rate limited by Zen.";
+        if (result.trigger === "ip" && result.ip) {
+          message += ` (Your IP: ${result.ip})`;
```

Comment:
escape? just to be sure

Reply:
The ip is already escaped inside the shouldBlockRequest function